cypress/e2e/login.spec.cy.js

```diff
@@ -0,0 +1,28 @@
+describe('Login and Profile Access', () => {
+  it('should log in and access profile', () => {
+    // Visit the website
+    cy.visit('https://kristiannilsenaug22ft.github.io/social-media-client/'); // Change the URL before PR
```

Comment:
```suggestion
    cy.visit('/');
```
You should be testing the code in this specific PR, not the code that is deployed to master. It's important that the files tested are the same files in the branch. Cypress will automatically run a web server for you and make this available at /